frontends/election-manager/src/config/types.ts

```diff
@@ -36,6 +36,12 @@ export const PrintableBallotType = {
 export type PrintableBallotType =
   typeof PrintableBallotType[keyof typeof PrintableBallotType];
 
+export enum BallotMode {
+  Official = 'live',
+  Test = 'test',
+  Sample = 'sample',
+}
```

Comment:
Thank you for the suggestion on unions, I was wondering which one would be better! 

I think I may want to make an exception here on the renaming. The literal value has to be 'live' in filenames, otherwise we might break the ballot package interpretation in `bsd`. However, we use `Official` in the UI so I think that it makes the code clearer. And with the enum, if we wanted to auto generate button text off of the enum, we could take the key value.

Thoughts?

Reply:
Got it! I had a feeling there might be a reason for the divergence - in that case, keeping `Official = 'live'` seems reasonable to me!